docs/06-configuration.md

```diff
@@ -62,6 +62,7 @@ Arguments passed to the CLI will always take precedence over the CLI options con
 - `require`: extra modules to require before tests are run. Modules are required in the [worker processes](./01-writing-tests.md#process-isolation)
 - `babel`: test file specific Babel options. See our [Babel recipe](./recipes/babel.md#configuring-babel) for more details
 - `babel.extensions`: extensions of test files that will be precompiled using AVA's Babel presets. Setting this overrides the default `"js"` value, so make sure to include that extension in the list
+- `timeouts`: Timeouts in AVA behave differently than in other test frameworks. AVA resets a timer after each test, forcing tests to quit if no new test results were received within the specified timeout. This can be used to handle stalled tests. See the [relevant tests](./test-timeouts.md#Test-timouts) for more timeout options.
```

Comment:
The option is `timeout` (singular).

The link is wrong, it should be for https://github.com/avajs/ava/blob/master/docs/07-test-timeouts.md. Rather than "tests" I'd say "See our [timeout documentation] for more options."